tools/rest_server/rest_app/test_execution_api.py

```diff
@@ -29,7 +29,7 @@
                 description='Test execution related operations')
 
 
-# pylint-disable-message=too-few-public-methods
+# pylint: disable=too-few-public-methods
```

Comment:
As disscussed, do we need this above all the functions ? As we have observed using once in function of same class, will disable the onwards same warning in whole file.

Reply:
Taken care